src/components/AdminContainer.js

```diff
@@ -140,103 +143,140 @@ class AdminContainer extends Component {
     return (
       <React.Fragment>
         {adminAddress ? (
-          <Panel>
-            {baseStyle}
-            <div className="flex">
-              <div className="w-50">
-                <h1 className="mt0">Admin</h1>
-              </div>
-              <div className="w-50">
-                <NetworkSelectorContainer />
-              </div>
-            </div>
-            <div className="flex bb pb3">
-              <div className="w-50">
-                <h3>
-                  Current account{" "}
-                  <div
-                    style={{ cursor: "pointer" }}
-                    className="dib click-to-refresh"
-                    onClick={this.refreshCurrentAddress}
-                    title="Try to grab current account"
-                    tabIndex={1}
-                  >
-                    <i className="fas fa-sync-alt" />
+          <React.Fragment>
+            <nav className="dt pa3 pl6-l pr6-l w-100 border-box ph5-ns bg-white shadow-1-ns items-center flex-l flex-row-l">
+              <a href="/" title="Home">
+                <img
+                  src={"../../static/images/logo.svg"}
+                  className="dtc v-mid mid-gray link dim pa3"
+                  style={{ width: "200px" }}
+                  alt="OpenCerts"
+                />
+              </a>
+              <div className="w-80-ns flex-l flex-row-l justify-end">
+                <div className="w-100-s pl2-l pr2-l">
+                  <h3 className="ma0">
+                    Current Account{" "}
+                    <div
+                      style={{ cursor: "pointer" }}
+                      className="dib click-to-refresh"
+                      onClick={this.refreshCurrentAddress}
+                      title="Try to grab current account"
+                      tabIndex={1}
+                    >
+                      <i className="fas fa-sync-alt" />
+                    </div>
+                  </h3>
+                  <div className="pa2">
+                    {adminAddress ? (
+                      <HashColor hashee={adminAddress} networkId={networkId} />
+                    ) : (
+                      <div className="red">No wallet address found.</div>
+                    )}
+                  </div>
+                </div>
+                <div className="w-100-s pl2-l pr2-l">
+                  <h3 className="ma0">Account Balance</h3>
+                  <div className="pa2">
+                    {accountBalance ? (
```

Comment:
if accountBalance is 0, then it will display `Unable to load account balance.`

Reply:
Updated. Changed to undefined when retrieving failed.

Reply:
it doesnt solve the issue, 0 is a falsy value

Reply:
Fixed.